src/gcs-resumable-upload.ts

```diff
@@ -27,6 +27,8 @@ import {GoogleAuth, GoogleAuthOptions} from 'google-auth-library';
 import {Readable, Writable} from 'stream';
 import retry = require('async-retry');
 import {RetryOptions, PreconditionOptions} from './storage';
+import * as packageJson from '../package.json';
```

Comment:
I didn't catch this line during our pairing, but I believe this may break [`package.json#files`](https://github.com/googleapis/nodejs-storage/blob/cbe2590186a6ef6229a10b7e62e3c4a1800dccf2/package.json#L13-L16) for consumers (via `npm i @google-cloud/storage`) as `build/package.json` wouldn't be included [[npm docs](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#files)].

We currently use this pattern for including `package.json` (a little messy): https://github.com/googleapis/nodejs-storage/blob/cbe2590186a6ef6229a10b7e62e3c4a1800dccf2/src/storage.ts#L617-L625

We have a few options:
- Perhaps we can add `build/package.json` to `package.json#files`. Drawback: some tooling may believe `build` in itself is a separate package (as `package.json`) and may misbehave.
- We can use the pattern found in `src/storage`. Drawback: messy-looking.

CC: @bcoe - as I'm working on an easy `package.json` import proposal for Node.js.